### ISSUE TITLE
Can't track link clicks with Google Tag Manager using id's (which is best practice)

### ISSUE THREAD

**Issue by DevineJasper** [soflyy/breakdance-bugs#786]
v1.2, Chrome, macOS

**Reproduction  Steps**

1. Add a button to Breakdance front-end editor
2. Add a ID to the button, for example 'header-reservation'

This results in the following code:
```
<div id="header-reservation" class="bde-button-323-120 bde-button" test="true">
    <a class="breakdance-link  button-atom button-atom--custom bde-button__button" href="#afspraak" target="_self" data-type="url">
       <span class="button-atom__text">Maak een afspraak</span>
    </a>
</div>
```

In Google Tag Manager, I am not able to track this event.
When clicking on the button, the following classes are sent with the event:
'breakdance-link  button-atom button-atom--custom bde-button__button', but no id's.

This means it's registering the click on the <a> tag.
However, the id is on the parent <div> element.

Is there any way to set the id on the <a> element instead of the container <div>?

**Comment by DevineJasper** [comment]
@a-dubiel Hi, this is a big issue for me.
As for now, I have to set tags based on the element's text.
However, if the client changes this text, the click isn't tracked anymore...
The solution would be: if you set an ID for a button, this is set on the <a> tag, not on the parent <div>

**Comment by a-dubiel** [comment]
@DevineJasper totally understandable. We just shipped this feature in the latest 1.3 RC. Stable release is imminent, I would say early next week. 

![image](https://github.com/soflyy/breakdance-bugs/assets/669243/cf997b99-c9a9-4a9a-ac67-91b5e5cca215)

**Comment by DevineJasper** [comment]
@a-dubiel awesome, beautiful integration. Another great implementation of this would be the ability to give every tab of an 'Advanced tabs' or just 'tabs' also the ability to set an ID!
